pkg/utils/informer/informermanager.go

```diff
@@ -213,6 +226,40 @@ func (s *informerManagerImpl) IsClusterScopedResources(gvk schema.GroupVersionKi
 	return resMeta.IsClusterScoped
 }
 
+func (s *informerManagerImpl) Remove(gvk schema.GroupVersionKind, registration cache.ResourceEventHandlerRegistration) {
```

Comment:
what is this for?

Reply:
This is where I was confusing myself. Originally, I thought I had to create/implement the remove function but then I looked when I worked/looked at the section above that's when I realized that we probably didn't need this since it would be removed above.